### ISSUE TITLE
blog: support multiple authors

### ISSUE THREAD

**Issue by casperdcl** [iterative/dvc.org#3502]
Many posts are really worked on a lot by multiple people. While we encourage one person to drive a post slightly more, it seems fair to include others as co-authors.

Basically can the YAML metadata block support `author: str-or-list`?

**Comment by jendefig** [comment]
Good point! Also it would show more faces in the company that are largely invisible now and drive home the collaboration aspect of this team sport....

**Comment by casperdcl** [comment]
Some ideas:

- separate: ![image](https://user-images.githubusercontent.com/10780059/166621011-a396188b-3a03-4fc3-9cd7-6ef93f5ef6e1.png)
- overlap+hover: ![image](https://user-images.githubusercontent.com/10780059/166621069-1fc0318e-42c0-4da8-bb1b-82706638963f.png) -> (hover) ![image](https://user-images.githubusercontent.com/10780059/166621085-dc4e4d07-544b-4269-b558-16a97cb699d6.png)

**Comment by jendefig** [comment]
I think it makes sense to do author and contributor(s) if that's possible.

**Comment by julieg18** [comment]
> I think it makes sense to do author and contributor(s) if that's possible.

Definitely possible! Instead of `author` into a list, we could add a parameter called `contributer`.

**Comment by casperdcl** [comment]
not sure, if we make distinctions then it'll probably have to be like a scientific paper where there could be multiple authors (i.e. joint-first-authors) and multiple contributors.

I think just maintaining the order of the author list is enough for now?

**Comment by jendefig** [comment]
> in the order they're found

what does this mean in the context of a blog post?

**Comment by jorgeorpinel** [comment]
Just that there's an implied "main author" (the first found in the frontmatter, shown first in the list).
